bangazonapi/views/product.py

```diff
@@ -104,6 +105,9 @@ def create(self, request):
 
             new_product.image_path = data
 
+        if float(new_product.price) > 17000.00:
```

Comment:
While this works, take a look at how you would use the validation fields on the model with the `clean_fields` method: https://docs.djangoproject.com/en/3.2/ref/models/instances/#validating-objects